go.mod

```diff
@@ -20,13 +20,15 @@ require (
 	github.com/elazarl/go-bindata-assetfs v1.0.0
 	github.com/fatih/color v1.7.0
 	github.com/fatih/structs v1.1.0
+	github.com/go-bindata/go-bindata v3.1.2+incompatible // indirect
```

Comment:
this is not needed as a dependency - please remove it. This PR shouldn't have any dependency additions.